pkg/platform/yaml/yaml.go

```diff
@@ -181,6 +181,32 @@ func ReadSessionIndices(_ context.Context, path string, Logger *zap.Logger) ([]s
 	return indices, nil
 }
 
+func ReadSessionFileIndices(_ context.Context, path string, Logger *zap.Logger) ([]string, error) {
+	var indices []string
+	dir, err := ReadDir(path, fs.FileMode(os.O_RDONLY))
+	if err != nil {
+		Logger.Debug("creating a folder for the keploy generated testcases", zap.Error(err))
+		return indices, nil
+	}
+
+	files, err := dir.ReadDir(0)
+	if err != nil {
+		return indices, err
+	}
+
+	for _, v := range files {
```

Comment:
Please try to unify it by passing an argument to the existing function of readsessionIndices